optaplanner-docs/pom.xml

```diff
@@ -42,52 +42,6 @@
 
     <pluginManagement>
       <plugins>
-        <plugin>
```

Comment:
What is the motivation for moving the asciidoc configuration from here to the optaplanner-parent?

Reply:
We agreed to move the <configuration> bit back into optaplanner-docs